.changeset/eleven-coats-compete.md

```diff
@@ -0,0 +1,7 @@
+---
+'nest-commander': patch
+'nest-commander-schematics': patch
+'nest-commander-testing': patch
```

Comment:
```suggestion
'nest-commander-schematics': patch
````

We should only need the patch for the schematics package. I don't think the other two need to be updated